source/postprocess/heat_flux_statistics.cc

```diff
@@ -61,6 +61,7 @@ namespace aspect
         boundary_indicators
           = this->get_geometry_model().get_used_boundary_indicators ();
         std::vector<double> local_values;
+        local_values.reserve(boundary_indicators.size());
         for (const auto p : boundary_indicators)
           local_values.push_back (local_boundary_fluxes[p]);
```

Comment:
Why doesn't this use emplace back here, since push_back was replaced by emplace back in a few other files?

Reply:
I think it might not trigger for type double as it doesn't matter there, but I will replace it as well.